Desktop/engine/enginesync.cpp

```diff
@@ -52,6 +51,12 @@ EngineSync::EngineSync(QObject *parent)
 	assert(!_singleton);
 	_singleton = this;
 	
+	_filterRunningResetTimer = new QTimer(this);
+	_filterRunningResetTimer->setInterval(2000);
```

Comment:
```suggestion
	_filterRunningResetTimer->setInterval(1000);
```

Reply:
emmm, 2s is most silly for me when I 'crazy' click filter haha, for 1s  I can feel slow but yes it's also enough. I recommended 2s before since most cases it can be separated from the delay of mouse click/doubleclick events. For the user, after he clicks the filter and returns to the analysis panel, 1s has passed😂

Reply:
I first started with 4s, but that was absolutely ridiculous :p